### PR TITLE
Master mdl 71576 65

### PR DESCRIPTION
**Pull request by safatshahin**
PR for #65

### CODE REVIEW

File: question/bank/viewquestionname/templates/questionname_display.mustache [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+{{#haslabel}}
+    <label for="{{labelfor}}" class="d-inline-flex flex-nowrap overflow-hidden w-100">
+        <span class="questionname flex-grow-1 flex-shrink-1 text-truncate">
+            {{namecontent}}
+        </span>
+        <span class="ml-1">
+            <span class="accesshide">{{#str}}idnumber, question{{/str}}</span><span class="badge badge-primary">{{idnumber}}</span>
+        </span>
+    </label>
+    <label for="{{labelfor}}" class="d-inline-flex flex-nowrap overflow-hidden w-100">
```
Comment (guillogo) [inline question/bank/viewquestionname/templates/questionname_display.mustache:57]: @safatshahin can you please add a `{{#tags}}` before this line here so we won't see the question like this:

image

Also the close one `{{/tags}}` at the end of `</label>`

**Review by guillogo — CHANGES_REQUESTED** [review]
Hi @safatshahin review done, added just 1 comment in the template. Thank you.

**Comment by safatshahin** [comment]
Hi @guillogo  
thanks for reviewing this, made the changes, please have a look now.

**Review by guillogo — APPROVED** [review]
Approved.

**Pull request merged**